src/test/java/reposense/util/StringsUtilTest.java

```diff
@@ -70,4 +73,17 @@ public void removeTrailingBackslash_emptyString_noChange() {
 
         Assert.assertEquals(emptyString, actualString);
     }
+
+    @Test
+    public void addQuotationMarksForPath_specialBashCharacters_success() {
+        Assume.assumeTrue(!SystemUtil.isWindows());
+        String specialBashSymbols = "!\"#$&'()*,;<=>?\\[]^`{| }";
```

Comment:
Perhaps we could link the relevant SO posts here and in `addQuotesForFilePath` for easy reference.

Reply:
Noted, will update this.

Reply:
I've updated it.

Reply:
Let's link the answer specifically: https://unix.stackexchange.com/a/357932

Reply:
Resolved.